From 9455df768984ec8dcdbe85017757ba50c2a4ffe7 Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Thu, 10 Oct 2019 18:47:59 +0000
Subject: [PATCH] OS-8020 fswatcher can't find its own errno Reviewed by: Jason
 King <jason.king@joyent.com> Approved by: Todd Whiteman
 <todd.whiteman@joyent.com>

---
 src/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Makefile b/src/Makefile
index 5125aceb..b7392529 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -64,6 +64,7 @@ SMARTDCLIBDIR =	$(DESTDIR)/smartdc/lib
 #
 bootparams :	LIBS +=		-ldevinfo
 bootparams :	WARN_FLAGS +=	-Wno-unused
+fswatcher :	CPPFLAGS +=	-D_REENTRANT
 fswatcher :	LIBS +=		-lthread -lnvpair -lavl
 zfs_recv :	LIBS +=		-lsocket
 zfs_send :	LIBS +=		-lsocket
-- 
2.17.2 (Apple Git-113)

